packages/runtime/src/runtime/menu.tsx

```diff
@@ -199,7 +219,17 @@ export const SideBarMenu: React.FC<MenuBaseProps> = ({ id, ...props }) => {
                 target={item.openNewTab ? "_blank" : "_self"}
                 to={item.page ? `/${item.page}` : String(item.url)}
               >
-                {!isCollapsed && item.label}
+                {!isCollapsed &&
+                  (item.customItem
+                    ? EnsembleRuntime.render([
+                        unwrapWidget(
+                          (selectedItem === item.page
+                            ? item.customItem.selectedWidget ||
+                              item.customItem.widget
+                            : item.customItem.widget) || {},
+                        ),
+                      ])
+                    : item.label)}
```

Comment:
Similar, can we move this into a function and use if/else instead of ternary for readability?